### PR TITLE
Update usage-analytics.md

### PR DESCRIPTION
**Pull request by haider392**
Microsoft 365 Usage Analytics is not available for government customers. If yes, please confirm from the Power BI Product Dev team and provide proper steps that work. Support has got several tickets from government customers complaining that they are not able to install the Microsoft 365 Usage Analytics and get data properly.

### CODE REVIEW

**Comment by haider392** [comment]
Please update the documentation appropriately

**Review by haider392 — COMMENTED** [review]
N/A

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/usage-analytics/usage-analytics.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ description: "Get an overview of how your organization is adopting Microsoft 365
 
 # Microsoft 365 usage analytics
 ---
-Microsoft 365 usage analytics is also available for Microsoft 365 US Government Community.
+Microsoft 365 usage analytics is currenty not available for Power BI Government Customers
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/usage-analytics.md:25]:
```suggestion
Microsoft 365 usage analytics is currently not available for Power BI Government customers.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Comment by mypil** [comment]
@haider392 - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Pull request merged**